Formula/g/gaul.rb

```diff
@@ -30,13 +30,12 @@ class Gaul < Formula
   end
```

Comment:
need to move out of the `on_macos` block here

Reply:
indeed 👍